keyboards/ergodox_infinity/rules.mk

```diff
@@ -1,32 +1,18 @@
 # MCU name
 MCU = MK20DX256
 
-# Linker script to use
-# - it should exist either in <chibios>/os/common/ports/ARMCMx/compilers/GCC/ld/
-#   or <this_dir>/ld/
-# - NOTE: a custom ld script is needed for EEPROM on Teensy LC
-# - LDSCRIPT =
-#   - MKL26Z64 for Teensy LC
-#   - MK20DX128 for Teensy 3.0
-#   - MK20DX256 for Teensy 3.1 and 3.2
-#   - MK20DX128BLDR4 for Infinity 60% with Kiibohd bootloader
-#   - MK20DX256BLDR8 for Infinity ErgoDox with Kiibohd bootloader
-MCU_LDSCRIPT = MK20DX256BLDR8
-
 # Vector table for application
 # 0x00000000-0x00001000 area is occupied by bootlaoder.*/
 # The CORTEX_VTOR... is needed only for MCHCK/Infinity KB
 OPT_DEFS += -DCORTEX_VTOR_INIT=0x00002000
```

Comment:
Wondering if this is needed anymore, does it relate to the `flash0` section in the ldscript being set to `256k - 0x2000` here? This board and the Infinity60 both have this line, the other two boards don't.

Reply:
I just tested the Infinity60 with Command disabled and with/without this line, I got the same MD5 in both cases. So it appears `CORTEX_VTOR_INIT` doesn't do anything anymore, and this commit seems to back that up: https://github.com/qmk/ChibiOS/commit/6ebca5756c4204bbb8f5205c9accc66f2aa65ed1#diff-8abf1b109d9e0fda18afbdd44140c4f8

Reply:
@tzarc @zvecr thoughts?

Reply:
Nuke it.